src/App.tsx

```diff
@@ -10,33 +12,23 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState<Tab>(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    setSelectedTab(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTab.title}`}
       </h1>
 
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
-        </div>
-
-        <div className="block" data-cy="TabContent">
-          Some text 1
-        </div>
-      </div>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
+        onTabSelected={onTabSelected}
```

Comment:
`onTabSelected={setSelectedTab}`

Reply:
it's better to pass only one number - `id` instead of object
<img width="799" alt="Screenshot 2023-02-08 at 10 51 53" src="https://user-images.githubusercontent.com/37452763/217481220-aa4a3c1c-6f34-4042-8d4d-9ee5102c2aaf.png">